include/RAJA/util/View.hpp

```diff
@@ -63,20 +64,24 @@
 namespace RAJA
 {
 
-template <typename DataType,
-          typename LayoutT,
-          typename DataPointer = DataType *>
+template <typename DATA_TYPE,
+          typename LAYOUT_TYPE,
+          typename DATA_POINTER = DATA_TYPE *>
 struct View {
-  LayoutT const layout;
+  using DataType = DATA_TYPE;
+  using DataPointer = DATA_POINTER;
+  using LayoutType = LAYOUT_TYPE;
```

Comment:
What does changing the template names to CAP_CASE with `using`s defined immediately at the top of View accomplish?

Reply:
Just as a style note, most places we use value_type, pointer_type, etc. to match up with std:: for this kind of thing (or are trying to move in that direction).  I wouldn't let that hold this up, but something to think about.